templates/scripts/pre-k8s.sh

```diff
@@ -33,16 +33,21 @@ fi
 
 kubectl --context ${KUBE_CONTEXT} -n user-auth get secrets ${PROJECT} > /dev/null 2>&1
 if [[ $? -ne 0 ]]; then
-	REGION=${AWS_DEFAULT_REGION} \
-	SEED=${RANDOM_SEED} \
-	PROJECT_NAME=${PROJECT} \
-	ENVIRONMENT=${ENVIRONMENT} \
-	NAMESPACE=user-auth \
-	DATABASE_TYPE=<% index .Params `database` %> \
-	DATABASE_NAME=user_auth \
-	USER_NAME=kratos \
-	CREATE_SECRET=secret-user-auth.yml.tpl \
-	sh ./create-db-user.sh
+    REGION=${AWS_DEFAULT_REGION} \
+    SEED=${RANDOM_SEED} \
+    PROJECT_NAME=${PROJECT} \
+    ENVIRONMENT=${ENVIRONMENT} \
+    NAMESPACE=user-auth \
+    DATABASE_TYPE=<% index .Params `database` %> \
+    DATABASE_NAME=user_auth \
+    USER_NAME=kratos \
+    CREATE_SECRET=secret-user-auth.yml.tpl \
+    <%- if ne (index .Params `sendgridApiKey`) "" %>
+    SMTP_URI=smtps://apikey:${sendgridApiKey}@smtp.sendgrid.net:465 \
```

Comment:
does this directly maps to kratos? 
in their docs it says [`COURIER_SMTP_CONNECTION_URI`](https://www.ory.sh/kratos/docs/reference/configuration)

Reply:
No, it's used in `secret-user-auth.yml.tpl `

Reply:
Yeah I don't think that prefix is useful, or maybe we leave it for flexibility but default it to `""`

Reply:
Ah i see, I assume kratos recognizes `smtpConnectionURI`? couldnt find it in their docs

Reply:
The secret we are creating matches the one created by the helm chart, though we are doing it manually rather than through the chart: https://github.com/ory/k8s/blob/master/helm/charts/kratos/templates/secrets.yaml

Reply:
ahh i see 👍  thanks for the explanation